docs/publishing.md

```diff
@@ -0,0 +1,26 @@
+# Publishing packages to NPM
+
+## Pre-requisites
+
+- npm credentials, Ona engineers should have access to this on bitwarden under the opensrp-web collection. This are only used in the publish step.
```

Comment:
You should remove the portion that talks about Ona engineers, I don't think it makes sense on this file.